input/test_cosmo-dd-multispecies.in

```diff
@@ -0,0 +1,105 @@
+include "input/test_cosmo-dd-fc0.in"
+
+
+Field {
+   list += ["HI_density",
+	   "HII_density",
+	   "HeI_density",
+	   "HeII_density",
+	   "HeIII_density",
+	   "e_density",
+           "metal_density",
+	   "cooling_time",
+	   "temperature",
+	   "gamma"];
+}
+
+Group {
+    list = ["color", "derived"];
+    color {
+       field_list = [
+              "HI_density",
+              "HII_density",
+              "HeI_density",
+              "HeII_density",
+              "HeIII_density",
+              "e_density",
+              "metal_density"];
+    }
+
+    derived {
+	field_list = ["temperature",
+		      "pressure",
+		      "cooling_time"];
+    }
+
+}
+
+Method {
+     list = [ "pm_deposit", "gravity", "ppm", "grackle", "pm_update", "comoving_expansion" ];
+    
+     output {
+        field_list += ["HI_density",
+	                 "HII_density",
+	                 "HeI_density",
+	                 "HeII_density",
+	                 "HeIII_density",
+	                 "e_density",
+                         "metal_density",
+	                 "cooling_time",
+	                 "temperature",
+	                 "gamma"]; 
```

Comment:
```suggestion
	                 "temperature"]; 
```
```suggestion
	   "temperature"];
```
Remove the `"gamma"` field since it isn't ever used (and to avoid confusing new users)